src/api/java/de/teamlapen/vampirism/api/entity/EntityClassType.java

```diff
@@ -35,4 +40,19 @@ public AttributeModifier getDamageModifier() {
     public AttributeModifier getSpeedModifier() {
         return speedModifier;
     }
+
+    public static int getID(EntityClassType entityclasstypes) {
+        return ENTITYCLASSTYPES.get(entityclasstypes);
+    }
+
+    public static EntityClassType getEntityClassType(int id) {
+        return ID.get(id);
```

Comment:
To be sure this should probably be checked against map.size()  and return a default if to large

Reply:
dont set an classtype if id is corrupt(entityclasstype is set by contructor)